Gruntfile.js

```diff
@@ -226,209 +132,183 @@ module.exports = function(grunt) {
-                    {
-                        expand: true,
-                        cwd: "src/static/",
-                        src: [
-                            "**/*",
-                            "**/.*",
-                            "!stats.txt",
-                            "!ga.html"
-                        ],
-                        dest: "build/dev/"
-                    }
+            webProd: {
+                target: "web",
+                entry: "./src/web/index.js",
+                output: {
+                    filename: "scripts.js",
+                    path: __dirname + "/build/prod"
+                },
+                plugins: [
+                    new webpack.optimize.UglifyJsPlugin({
+                        compress: {
+                            "screw_ie8": true,
+                            "dead_code": true,
+                            "unused": true,
+                            "warnings": false
+                        },
+                        comments: false,
+                    }),
+                    new HtmlWebpackPlugin({ // Main version
+                        filename: "index.html",
+                        template: "./src/web/html/index.html",
+                        compileTime: compileTime,
+                        minify: {
+                            removeComments: true,
+                            collapseWhitespace: true,
+                            minifyJS: true,
+                            minifyCSS: true
+                        }
+                    }),
+                    new HtmlWebpackPlugin({ // Inline version
+                        filename: "cyberchef.htm",
+                        template: "./src/web/html/index.html",
+                        compileTime: compileTime,
+                        inline: true,
+                        minify: {
+                            removeComments: true,
+                            collapseWhitespace: true,
+                            minifyJS: true,
+                            minifyCSS: true
+                        }
+                    }),
                 ]
             },
-            staticProd: {
-                files: [
-                    {
-                        expand: true,
-                        cwd: "src/static/",
-                        src: [
-                            "**/*",
-                            "**/.*",
-                            "!stats.txt",
-                            "!ga.html"
-                        ],
-                        dest: "build/prod/"
-                    }
-                ]
+            tests: {
+                target: "node",
+                entry: "./test/index.js",
+                output: {
+                    filename: "index.js",
+                    path: __dirname + "/build/test"
+                }
             },
+            node: {
+                target: "node",
+                entry: "./src/node/index.js",
+                output: {
```

Comment:
Should use https://www.npmjs.com/package/webpack-node-externals so that the node build doesn't contain all of the 3rd party deps